pyevtk/hl.py

```diff
@@ -184,9 +224,11 @@ def imageToVTK(
 
 
 # ==============================================================================
-def gridToVTK(path, x, y, z, cellData=None, pointData=None, fieldData=None):
+def gridToVTK(
+    path, x, y, z, start=(0, 0, 0), cellData=None, pointData=None, fieldData=None
```

Comment:
Same as `imageToVTK`

Reply:
I don't think that moving this arguments after `fieldData` makes much sense.